charts/misc-apps/examples/reloader.yaml

```diff
@@ -0,0 +1,10 @@
+# reloader watching namespaces labeled with reloader=true and exporting metrics
+
+reloader:
+  enabled: true
+  project: infra-reloader
+  values:
+    reloader:
+      namespaceSelector: "reloader=true"
```

Comment:
is this really a string? why doesn't it use k8s api style specs?

Reply:
6213f5c

Reply:
is reloader just a kubectl wrapper? wouldn't a real k8s api client specify it as follows:
```yaml
namespaceSelector:
  reloader: "true"
```

Reply:
You are right. But the client expects this:

https://github.com/stakater/Reloader/blob/master/README.md?plain=1#L217

and the value is just passed over as argument in the deployment so...